toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -155,11 +155,11 @@ repo_branch=$(git -C ${repo_path} rev-parse --abbrev-ref HEAD)
 # Generate text based on mode (Use figlet to generate splash text once available on Azure Linux)
 if [[ "${mode}" == "build" ]]; then
     echo -e "\033[31m -----------------------------------------------------------------------------------------\033[0m" > ${tmp_dir}/splash.txt
-    echo -e "\033[31m ----------------------------------- MARINER BUILDER ! ----------------------------------- \033[0m" >> ${tmp_dir}/splash.txt
+    echo -e "\033[31m --------------------------------- AZURE LINUX BUILDER ! -------------------------------- \033[0m" >> ${tmp_dir}/splash.txt
     echo -e "\033[31m -----------------------------------------------------------------------------------------\033[0m" >> ${tmp_dir}/splash.txt
 else
     echo -e "\033[31m -----------------------------------------------------------------------------------------\033[0m" > ${tmp_dir}/splash.txt
-    echo -e "\033[31m ----------------------------------- MARINER TESTER ! ------------------------------------ \033[0m" >> ${tmp_dir}/splash.txt
+    echo -e "\033[31m --------------------------------- AZURE LINUX TESTER ! --------------------------------- \033[0m" >> ${tmp_dir}/splash.txt
```

Comment:
supernit: Looks like you're missing a `-` at the end of this string, if we want to match up with the others near it

```suggestion
    echo -e "\033[31m --------------------------------- AZURE LINUX TESTER ! ---------------------------------- \033[0m" >> ${tmp_dir}/splash.txt
```